packages/velo-external-db-core/lib/service/data.spec.js

```diff
@@ -168,6 +171,8 @@ describe('Data Service', () => {
         ctx.itemId = chance.guid()
         ctx.itemIds = Array.from({ length: 10 }, () => chance.guid())
         ctx.total = chance.natural({ min: 2, max: 20 })
+        ctx.field = gen.randomDbField()
```

Comment:
Generate random column, this column will be the only column in the schema that will be return be the mocked 
`CacheableSchemaInformation`